net/mwan3/files/lib/mwan3/mwan3.sh

```diff
@@ -273,48 +276,49 @@ mwan3_set_connected_ipv4()
 
 mwan3_set_connected_iptables()
 {
-	local connected_network_v6 source_network_v6
+	local connected_network_v6
```

Comment:
Why did you remove the local variable declaration **source_network_v6**

Reply:
thanks for catching. `source_network_v6` was a casualty of the recent rebase. He has been resuscitated in 54f41ca54080f343ea787589fe37d957f962ff6c